### ISSUE TITLE
Possible problem with nfs-utils 

### ISSUE THREAD

**Issue by WetGeek** [getsolus/packages#1150]
### Summary

My network at home includes a Synology DiskStation415 NAS that my hardware and virtual machines access with Solus.  It's an enterprise-level server that all the machines here can access via nfs.  The Solus systems use nfs-utils to access the shares that are mounted in /mnt.

This morning, my daily-driver laptop was unable to access the ISOs share on the NAS when I tried to use Etcher to create an installer flash drive to test a Solus issue.  I checked the NAS and found it not active.  When I restarted it, I was no longer able to access any of its shares via their mount points in /mnt.  Strangely enough, though, I was able to use Dolphin to browse the Network, and access the server and all its shares via Dolphin's default SMB protocol.  I was able to determine that the NAS was healthy, and all of its contents were still there and available via SMB.

My first thought was to try a different distro, to see whether it was a Synology issue or a Solus issue.  I installed Sparky Linux on a spare testing laptop, and configured it far enough to attempt to connect to the NAS using nfs.  Sparky is a rolling distro based on Debian, thus it wouldn't use any system software that's also used by Solus.  In particular, Sparky uses nfs-common to access that server instead of Solus' nfs-utils.  The result of this was no problem accessing those shares and their contents using Dolphin.  

So, the bottom line is that if Solus isn't used, the problem disappears.  Another distribution using different software to acess the NAS server has no problem retrieving nfs data.  Even Solus, using SMB is able to access it all.  Solus, using nfs-utils cannot do that.  That has caused me to believe that there's a problem with Solus' implementation of nfs-utils.  The mount points in /mnt are empty after a call to mount -a. 

### Steps to reproduce

Because reproducing this would require access to a similar NAS server with shares that can be accessed via nfs, it's probably a better idea to investigate whether anything might be wrong with the Solus implementation of nfs-utils.  Has it recently changed, etc.?

### Expected result

I expected to be able to access my NAS shares as I do on a daily basis.  They're even linked to the folders in my /home directory, so that access to Documents, Downloads, Music, Pictures, Videos,  and so forth actually access nfs shares instead of local folders.

### Actual result

Neither Dolphin nor any other applications are able to access the NAS shares as of this morning.  To the best of my knowledge, they worked all day yesterday.  And I don't know of any events that might have affected that access.  The NAS was shut down and restarted, and still unable to access any shares using nfs.

### Environment

- [X] Is system up to date?

### Repo

Shannon (stable)

### Desktop Environment

Plasma

### System details

![image](https://github.com/getsolus/packages/assets/6805994/e08109ac-5a7b-4d22-9440-5d1af9df14f1)

**Comment by WetGeek** [comment]
I continued to configure the Sparky laptop so it's now an exact duplicate of my daily-driver laptop, so I can temporarily replace that laptop with this one if necessary.

**Comment by WetGeek** [comment]
I checked before I entered this report, and nfs access to my NAS was still not working.  After I posted the report, I went to the forum and found out that the sync has been done, and upgrades are available.

The first thing I do before an udate is Restic backup, which goes to a repository on a nfs server share.  And it worked.  So just as nfs access failed this morning, and was still unavailable when I filed this report, it is now working, so I'm going to close this issue now.  I can reopen it if the problem presents itself again.
